docs/reference/slm/apis/slm-start.asciidoc

```diff
@@ -0,0 +1,51 @@
+[[slm-api-start]]
+=== Start snapshot lifecycle management API
+
+[subs="attributes"]
+++++
+<titleabbrev>Start snapshot lifecycle management</titleabbrev>
+++++
+
+Turn on snapshot lifecycle management (SLM).
+
+[[slm-api-start-request]]
+==== {api-request-title}
+
+`POST /_slm/start`
+
+[[slm-api-start-prereqs]]
+==== {api-prereq-title}
+
+* If the {es} {security-features} are enabled, you must have the `manage_slm`
+cluster privilege to use this API. For more information, see
+<<security-privileges>>.
+
+[[slm-api-start-desc]]
+==== {api-description-title}
+
+Starts the SLM plugin if it is not running. 
+SLM starts automatically when a cluster is formed. 
+Manually starting SLM is only necessary if it has been stopped using the <<slm-api-stop>>.
+
+==== Request Parameters
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=timeoutparms]
+
+[[slm-api-start-example]]
+==== {api-examples-title}
+
+Start the SLM plugin:
+
+[source,console]
+--------------------------------------------------
+POST _slm/start
+--------------------------------------------------
+
+If successful, this request returns:
+
+[source,console-result]
+--------------------------------------------------
+{
+  "acknowledged": true
+}
+--------------------------------------------------
```

Comment:
I don't think this acknowledged response is needed.

Reply:
Agreed.